crates/sozo/src/commands/events.rs

```diff
@@ -29,6 +34,10 @@ pub struct EventsArgs {
     #[arg(help = "Continuation string to be passed for rpc request")]
     pub continuation_token: Option<String>,
 
+    #[arg(long)]
+    #[arg(help = "Print values as raw json")]
+    pub json: bool,
```

Comment:
once we move to scarb ui, this option will be handled by the logger

Reply:
i think we will still need a way to configure it in `SozoArgs` instead, for now we have hard coded output format to be text.

we need to roll out support for json output on all commands at same time, i tried using JSON mode for `sozo migrate` subcommand and it prints nothings.